Newsletter link in README (#747)
asreview/asreview: README.md

  the performance of the active learning software on the results of fully labeled systematic
  reviews. To use the simulation mode, knowledge on programming and bash/Command Prompt
  is highly recommended.
  
Want to keep up-to-date with the latest ASReview updates?

> [Sign up for the newsletter 📰](http://asreview.ai/newsletter)

Reviewer: ```suggestion
[Sign up for the newsletter 📰](https://asreview.ai/newsletter)
```

Instructions for usage with Docker are [here](docker/README.md).

> Want to keep up-to-date with the latest version?

Reviewer: ```suggestion
Want to keep up-to-date with the latest ASReview updates?
```

Instructions for usage with Docker are [here](docker/README.md).

> 

Reviewer: ```suggestion
```

contact with the team are:

- Documentation - [asreview.readthedocs.io](https://asreview.readthedocs.io/)

> - Newsletter - [asreview.ai/newsletter](http://asreview.ai/newsletter)

Reviewer: ```suggestion
- Newsletter - [asreview.ai/newsletter](https://asreview.ai/newsletter)
```
